build.sh

```diff
@@ -466,13 +468,21 @@ case "$COMMAND" in
 
     "test-ios-devices")
         failed=0
-        test_ios_devices xcrealm "iOS Device Tests" "$CONFIGURATION" || failed=1
-        if [ $REALM_SWIFT_VERSION != '1.2' ]; then
-            test_ios_devices xcrealmswift "RealmSwift" "$CONFIGURATION" || failed=1
-        fi
+        sh build.sh test-ios-devices-objc || failed=1
+        sh build.sh test-ios-devices-swift || failed=1
         exit $failed
```

Comment:
It could be marginally cleaner to write this as:

    failed=0
    trap "failed=1" ERR
    sh build.sh test-ios-devices-objc
    sh build.sh test-ios-devices-swift
    exit $failed